web/src/lib/components/shared-components/profile-image-cropper.svelte

```diff
@@ -0,0 +1,55 @@
+<script lang="ts">
+  import { AssetResponseDto, api } from '@api';
+  import { createEventDispatcher } from 'svelte';
+  import PhotoViewer from '../asset-viewer/photo-viewer.svelte';
+  import BaseModal from './base-modal.svelte';
+  import Button from '../elements/buttons/button.svelte';
+  const dispatch = createEventDispatcher();
```

Comment:
I don't like mixed imports and general statements. There should be a big block of imports and then the application code.